plugins/dicom_viewer/webpack.helper.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(x) {
```

Comment:
Let's name this something more descriptive than `x`, like `webpackConfig` perhaps.